Upgrade from 1.7.3.ynh1 to 1.7.3.ynh2 contains failed dependencies

Upgrading to the latest grist package along with the YunoHost update to 12.1.17.1 (stable) results in:

2025-09-06 14:38:45.260 - debug: 3-pipe Sandbox started sandboxPid=1716355, flavor=pyodide, command=undefined, entryPoint=(default), docId=2u5dhzqo7BZGgmPcYwro>
: 2025-09-06 14:38:45.280 - info: Sandbox stderr: node:internal/modules/cjs/loader:1386
:   throw err;
:   ^
: Error: Cannot find module './_build/worker/node_modules/pyodide'
: Require stack:
: - /var/www/grist/sandbox/pyodide/pipe.js
:     at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
:     at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
:     at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
:     at Function._load (node:internal/modules/cjs/loader:1192:37)
:     at TracingChannel.traceSync (node:diagnostics_channel:322:14)
:     at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)
:     at Module.require (node:internal/modules/cjs/loader:1463:12)
:     at require (node:internal/modules/helpers:147:16)
:     at Object.<anonymous> (/var/www/grist/sandbox/pyodide/pipe.js:4:25)
:     at Module._compile (node:internal/modules/cjs/loader:1706:14) {
:   code: 'MODULE_NOT_FOUND',
:   requireStack: [ '/var/www/grist/sandbox/pyodide/pipe.js' ]
: }

--- a/patches/main/setup.sh.patch
+++ b/patches/main/setup.sh.patch
@@ -2,7 +2,7 @@ diff --git a/sandbox/pyodide/setup.sh b/sandbox/pyodide/setup.sh
 index 7fe46563..7991b920 100755
 --- a/sandbox/pyodide/setup.sh
 +++ b/sandbox/pyodide/setup.sh
-@@ -8,8 +8,9 @@ echo "## Get pyodide node package"
+@@ -8,8 +8,10 @@ echo "## Get pyodide node package"
  
  if [[ ! -e _build/worker ]]; then
    mkdir -p _build/worker
@@ -13,6 +13,7 @@ index 7fe46563..7991b920 100755
 +  pushd _build/worker
 +    yarn init --yes
 +    touch yarn.lock
++    echo 'nodeLinker: "node-modules"' > ".yarnrc.yml"
 +    yarn add pyodide@0.23.4
 +  popd
  fi

